CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Master
+
+* Adds initializing android sdk from using `Instabug.start` instead of inside the application class
```

Comment:
Same as [this](https://github.com/Instabug/Instabug-React-Native/pull/356/files#r311559176). Any migration steps needed?